src/Hooks/Hook.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Carnival\Hooks;
+
+use Carnival\Hooks\Event;
+use Illuminate\Support\Collection;
+
+class Hook {
+
+    protected Collection $hooks;
+
+    public function __construct() {
+        $this->hooks = collect();
+    }
+
+    function add (string $eventClass, callable $callback) : void {
```

Comment:
[Consistency Police] Sometimes you write `functionName ()` and sometimes `functionName()`.
Latter is strongly preferred 🙃

Same in other files

Reply:
Okay, I can change this.